app/api/events/past/route.ts

```diff
@@ -13,8 +13,7 @@ export async function GET(req: NextRequest) {
     status: 200,
     headers: {
       "Content-Type": "application/json",
-      "Access-Control-Allow-Origin":
-        "https://sapeoplestagjthgmptzb46i.z8.web.core.windows.net",
+      "Access-Control-Allow-Origin": "*",
```

Comment:
Don’t use *
Add the actual urls

Reply:
See comment https://github.com/SSWConsulting/SSW.Website/pull/2866#discussion_r1695130398